includes/class-newspack-ads-gam.php

```diff
@@ -283,7 +283,17 @@ public static function get_serialised_gam_ad_units( $ids = [] ) {
 			}
 			return $ad_units_serialised;
 		} catch ( \Exception $e ) {
-			return [];
+			$error_message = $e->getMessage();
+			if ( 0 < strpos( $error_message, 'NETWORK_API_ACCESS_DISABLED' ) ) {
```

Comment:
```suggestion
			if ( 0 <= strpos( $error_message, 'NETWORK_API_ACCESS_DISABLED' ) ) {
```

If the message is at position 0 it should also be read as the same error.